LDK/test/src/org/labkey/test/util/external/labModules/LabModuleHelper.java

```diff
@@ -426,4 +434,34 @@ public void initiatePipelineJob(String importAction, List<String> files, String
 
         _test._fileBrowserHelper.selectImportDataAction(importAction);
     }
+
```

Comment:
Same comment as originally posted: does this viral load assay code belong in LabModuleHelper, or can it live in ViralLoadAssayTest?  Where's it get used?  I dont see reference to 'viral' in LabModulesTest or LabModuleHelper in 20.7.

Reply:
I just moved the now-working code into ViralLoadAssayTest

Reply:
thanks